database-seed/remote-image-getter.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs');
+const path = require('path');
+const axios = require('axios');
+
+const uri = 'https://loremflickr.com/';
+const productTypes = ['jewellery','dress','hat','cargopants','craftproduct','kitten','necklace','purse','shoes','chair','socks','computer','toys','collectible'];
+const resolutions = [[800,600],[600,600],[1200,800],[800,1200],[300,400],[200,150]];
+
+const generateImages = (num) => {
+    for (let i = 0; i < num; i++) {
+        let prodID = `${i}`;
+        let destination = path.join(__dirname, `/images/${prodID}/`);
+        let filename = `${prodID}.jpg`;
+        let resolution = randResolution();
+        let type = randType();
+        let url = uri+resolution+type;
+
+        try {
+            fs.statSync(destination).isDirectory();
+        } catch (error) {
+            if (error.code === 'ENOENT') {
+                fs.mkdirSync(destination);
+            } else {
+                throw error;
+            }
+        }
+
+        axios({
+            method: 'GET',
+            url: url,
+            responseType: 'stream'
+        })
+          .then((res) => {
+              console.log(`Image connection success for ${url}, processing stream to file at ${destination+filename}`);
+              res.data.pipe(fs.createWriteStream(destination+filename));
```

Comment:
so are you creating a directory with the name of `prodID` and then within that directory also a file with the name of `prodID` ?